cpp/src/arrow/compute/kernels/minmax.cc

```diff
@@ -95,8 +101,8 @@ class MinMaxAggregateFunction final
       }
       reader.Next();
     }
```

Comment:
Not your mistake, but could you please fix: `array.null_bitmap_data()` can return null, in which case `BitmapReader` will segfault. This can be an if/else the null_count, only constructing the BitmapReader if null_count > 0.

Reply:
done